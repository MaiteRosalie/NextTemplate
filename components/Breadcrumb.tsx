import React, { useState, useRef, useEffect } from 'react';
import styled from '@emotion/styled';
import { Box, Image, Flex, Text } from 'rebass';

const Wrapper = styled(Flex)`
  position: relative;
  overflow: hidden;
`;

const Background = styled(Box)`
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  object-fit: cover;
  z-index: -1;
  img {
    width: 100%;
    height: 100%;
    object-fit: cover;
    filter: brightness(0.5);
  }
`;
export type BreadCrumbProps = {
  src: string;
  title: string;
};

export const BreadCrumb = ({ title, src }: BreadCrumbProps) => {
  const [animated, setAnimated] = useState(false);
  const imageRef = useRef<HTMLImageElement>(null);

  const onLoad = () => {
    setAnimated(true);
  };

  useEffect(() => {
    if (imageRef.current && imageRef.current.complete) {
      setAnimated(true);
    }
  }, []);
  return (
    <Wrapper
      as="header"
      alignItems="center"
      justifyContent="center"
      p={3}
      pt="5rem"
      minHeight={['40vh']}
    >
      <Background as="picture">
        <source srcSet={`./static/images/${src}-1500.png`} media="(min-width: 800px)" />
        <source srcSet={`./static/images/${src}-800.png`} media="(max-width: 800px)" />
        <Image
          src={`./static/images/${src}-1500.png`}
          alt={title}
          onLoad={onLoad}
          ref={imageRef}
          css={{
            transition: `opacity 300ms cubic-bezier(0.4, 0, 0.2, 1),
            transform 300ms cubic-bezier(0.4, 0, 0.2, 1)`,
            opacity: animated ? '1' : '0',
          }}
        />
      </Background>
      <Flex
        width={1}
        flexDirection="column"
        alignItems="center"
        justifyContent="center"
        css={{
          transition: `opacity 600ms cubic-bezier(0.4, 0, 0.2, 1) 150ms,
          transform 600ms cubic-bezier(0.4, 0, 0.2, 1) 150ms`,
          opacity: animated ? '1' : '0',
          transform: animated ? 'translateY(-1rem)' : 'translateY(0rem)',
        }}
      >
        <Text as="h1" color="white" fontSize={[5, 6]} fontFamily="heading" textAlign="center">
          {title}
        </Text>
      </Flex>
    </Wrapper>
  );
};
