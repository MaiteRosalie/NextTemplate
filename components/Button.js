import React from 'react';
import { Button as BaseButton } from 'rebass';

export const Button = ({ disabled, children, rounded, ...props }) => {
  let tag;
  tag = props.href ? 'a' : 'button';
  tag = tag === 'a' && disabled ? 'span' : tag;

  return (
    <BaseButton
      sx={{
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
        borderRadius: rounded ? '30px' : '4px',
      }}
      as={tag}
      disabled={disabled && tag === 'button' ? disabled : undefined}
      {...props}
    >
      {children}
    </BaseButton>
  );
};
