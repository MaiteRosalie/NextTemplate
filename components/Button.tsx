import React from 'react';
import { Button as BaseButton, ButtonProps as BaseButtonProps } from 'rebass';

type ButtonProps = Pick<
  BaseButtonProps,
  Exclude<keyof BaseButtonProps, 'css' | 'as' | 'variant' | 'disabled'>
> & {
  rounded?: boolean;
};

export const Button = ({ children, rounded, ...props }: ButtonProps) => {
  return (
    <BaseButton
      sx={{
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
        borderRadius: rounded ? '30px' : '2px',
      }}
      as={props.href ? 'a' : 'button'}
      {...props}
    >
      {children}
    </BaseButton>
  );
};
