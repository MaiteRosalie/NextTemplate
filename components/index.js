export * from "./Layout";
export * from "./Modal";
export * from "./Button";
export * from "./Navigation";
export * from "./FancyText";
