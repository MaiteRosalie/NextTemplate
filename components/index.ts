export * from './Layout';
export * from './Modal';
export * from './ModalContext';
export * from './Button';
export * from './Navigation';
export * from './FancyText';
export * from './Navigation';
export * from './Carousel';
export * from './Header';
export * from './Features';
export * from './Steps';
export * from './CTA';
export * from './Footer';
export * from './Testimonials';
