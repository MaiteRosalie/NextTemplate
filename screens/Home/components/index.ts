export * from './Header';
export * from './Features';
export * from './Steps';
export * from './CTA';
export * from './Testimonials';
export * from './Icons';
