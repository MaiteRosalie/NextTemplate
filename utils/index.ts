export * from './tests';
export * from './devices';
export * from './location';
