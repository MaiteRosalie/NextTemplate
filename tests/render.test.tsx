import React from 'react';

import { render } from '../utils/tests';
import { Footer, Header, Layout, Message, Modal, Steps, Features } from '../components';

describe('render as espected', () => {
  test('Footer', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
  test('Header', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
  test('Layout', () => {
    const { container } = render(<Layout> hi </Layout>);
    expect(container).toMatchSnapshot();
  });
  test('Message', () => {
    const { container } = render(<Message />);
    expect(container).toMatchSnapshot();
  });
  test('Modal', () => {
    const { container } = render(<Modal isOpen> hi </Modal>);
    expect(container).toMatchSnapshot();
  });
  test('Steps', () => {
    const { container } = render(<Steps />);
    expect(container).toMatchSnapshot();
  });
  test('Features', () => {
    const { container } = render(<Features />);
    expect(container).toMatchSnapshot();
  });
});
