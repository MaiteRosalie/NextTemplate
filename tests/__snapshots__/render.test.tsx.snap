// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render as espected Features 1`] = `
<div>
  <section
    class="css-vurnku"
  >
    <div
      class="css-18e384y"
    >
      <div
        class="css-xn3tps-Item e1jrehui2"
      >
        <div
          class="css-1thwfmj-Icon e1jrehui0"
        >
          <svg
            class="css-r4avzq-Animated e1jrehui1"
            version="1.1"
            viewBox="0 0 13.229166 13.229166"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <title>
              Timer icon
            </title>
            <circle
              cx="6.6146"
              cy="7.2644"
              r="5.6226"
            />
            <circle
              cx="6.5581"
              cy="7.2362"
              r="4.0121"
            />
            <circle
              cx="6.5863"
              cy="7.2644"
              r=".59334"
            />
            <path
              d="m6.6146 11.197v-1.0454"
            />
            <path
              d="m4.4978 5.1194-0.73922-0.73922"
            />
            <path
              d="m9.4423 10.064-0.73922-0.73922"
            />
            <path
              d="m8.4771 5.2607 0.73922-0.73922"
            />
            <path
              d="m3.8434 9.9509 0.73922-0.73922"
            />
            <path
              d="m9.4824 7.2927h1.0454"
            />
            <path
              d="m2.6731 7.2644h1.0454"
            />
            <path
              d="m6.5581 6.5722v-2.1473"
            />
            <rect
              height="1.2997"
              rx="0"
              ry=".33905"
              width="1.8365"
              x="5.6257"
              y=".34213"
            />
          </svg>
        </div>
        <h4
          class="css-103ftbx"
        >
          Quick approval rates
        </h4>
        <p
          class="css-nibnrh"
        >
          Funding is now faster and simpler than ever.
        </p>
      </div>
      <div
        class="css-xn3tps-Item e1jrehui2"
      >
        <div
          class="css-1thwfmj-Icon e1jrehui0"
        >
          <svg
            class="css-r4avzq-Animated e1jrehui1"
            version="1.1"
            viewBox="0 0 13.229166 13.229166"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <title>
              Money icon
            </title>
            <g
              transform="translate(0 -283.77)"
            >
              <circle
                cx="6.6146"
                cy="290.39"
                r="6.1013"
              />
              <path
                d="m6.5969 294.48v-1.1437"
              />
              <path
                d="m8.0138 287.62-1.8095 0.0341s-1.3656-0.0342-1.4168 1.3827c0.068282 1.7241 1.7753 1.3656 1.7753 1.3656s1.9431-0.0507 1.8777 1.3827c-0.058308 1.2794-1.0413 1.4169-1.8607 1.4169h-1.451"
              />
              <path
                d="m6.5969 287.44v-1.1437"
              />
            </g>
          </svg>
        </div>
        <h4
          class="css-103ftbx"
        >
          Flexible payment plans
        </h4>
        <p
          class="css-nibnrh"
        >
          We give you ultimate payment flexibility.
        </p>
      </div>
      <div
        class="css-xn3tps-Item e1jrehui2"
      >
        <div
          class="css-1thwfmj-Icon e1jrehui0"
        >
          <svg
            class="css-r4avzq-Animated e1jrehui1"
            version="1.1"
            viewBox="0 0 13.229166 13.229166"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <title>
              Credit card icon
            </title>
            <rect
              height="7.2906"
              rx=".89769"
              ry=".87407"
              width="11.259"
              x=".98491"
              y="2.9693"
            />
            <path
              d="m0.98038 4.4412h11.268"
            />
            <path
              d="m0.98038 5.835h11.268"
            />
            <path
              d="m2.386 7.843h2.9766"
            />
            <path
              d="m2.386 8.8352h2.9766"
            />
            <rect
              height="1.4324"
              rx=".40672"
              ry=".40672"
              width="1.4324"
              x="9.3356"
              y="7.3513"
            />
            <path
              d="m0.68508 11.032 12.213-8.7879"
            />
          </svg>
        </div>
        <h4
          class="css-103ftbx"
        >
          No credit score required
        </h4>
        <p
          class="css-nibnrh"
        >
          Support, makes your experience faster and more accurate than ever.
        </p>
      </div>
      <div
        class="css-xn3tps-Item e1jrehui2"
      >
        <div
          class="css-1thwfmj-Icon e1jrehui0"
        >
          <svg
            class="css-r4avzq-Animated e1jrehui1"
            version="1.1"
            viewBox="0 0 13.229166 13.229166"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <title>
              Paperwork icon
            </title>
            <path
              d="m3.0474 12.591v-10.772h5.5043l2.8703 2.8703v7.9257z"
            />
            <path
              d="m2.8348 11.552h-0.8032v-10.796h8.3863v2.9293"
            />
            <path
              d="m8.5753 1.7954v2.8112h2.764"
            />
            <path
              d="m4.3703 10.725h5.1027"
            />
            <path
              d="m4.3703 7.4414h5.1027"
            />
            <path
              d="m4.3703 5.7996h5.1027"
            />
            <path
              d="m4.3703 4.1577h3.0213"
            />
            <path
              d="m4.3703 9.0832h5.1027"
            />
          </svg>
        </div>
        <h4
          class="css-103ftbx"
        >
          Minimal Paperwork
        </h4>
        <p
          class="css-nibnrh"
        >
          Tools are beyond the one-click install, async tech merges with ultimate flexibility.
        </p>
      </div>
    </div>
  </section>
</div>
`;

exports[`render as espected Footer 1`] = `
<div>
  <div
    class="css-jnc4ax-Wrapper e18f8y721"
  >
    <div
      class="css-1tar3ql"
    >
      <div
        class="css-1bqr18x"
      >
        <img
          alt="McAfee SECURE"
          class="css-1uqn0ab-Footer"
          src="./static/images/mcafee.png"
        />
        <a
          class="e18f8y720 css-3euuug-HL"
          href="./"
        >
          Apply Now
        </a>
        <a
          class="e18f8y720 css-3euuug-HL"
          href="./"
        >
           What is MCA?
        </a>
        <a
          class="e18f8y720 css-3euuug-HL"
          href="./"
        >
          About us
        </a>
        <a
          class="e18f8y720 css-3euuug-HL"
          href="./"
        >
          Contact
        </a>
        <a
          class="e18f8y720 css-3euuug-HL"
          href="./"
        >
          Login
        </a>
      </div>
      <div
        class="css-ipt8zy"
      >
        <a
          class="e18f8y720 css-1smhtko-HL"
          href="https://www.instagram.com/gldfunding/"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
            />
          </svg>
        </a>
        <a
          class="e18f8y720 css-1smhtko-HL"
          href="https://www.facebook.com/GLD-Funding-102567247876620/"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 320 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
            />
          </svg>
        </a>
        <a
          class="e18f8y720 css-1smhtko-HL"
          href="./"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
            />
          </svg>
        </a>
        <a
          class="e18f8y720 css-1smhtko-HL"
          href="./"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 576 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
            />
          </svg>
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`render as espected Header 1`] = `
<div>
  <header
    class="css-1urhh2l-Wrapper eiscj4c0"
  >
    <img
      alt="Hero background"
      class="fadeIn eiscj4c1 css-10ie8fl-Background"
      src="./static/images/bg-3.jpeg"
    />
    <div
      class="css-339bp5"
    >
      <div
        class="css-10gcz7t"
      >
        Funding Solutions for your business
      </div>
      <h1
        class="css-zmrqg7"
      >
        We'll Help Your Business 
         
        <div
          class="css-86bk71-Wrapper e1jkelvh0"
        >
          <span
            class="css-iifya3-FancyText"
          >
            Succeed
          </span>
          <div
            class="css-1zk1a-Animated-FancyText e1jkelvh1"
          >
            Succeed
          </div>
          <div
            class="css-ml3iyx-Animated-FancyText e1jkelvh1"
          >
            Grow
          </div>
        </div>
      </h1>
      <form
        class="css-mn4t1z-Form eiscj4c2"
      >
        <input
          inputmode="numeric"
          placeholder="Requested Amount"
          required=""
          type="text"
          value=""
        />
        <button
          class="css-1nviw9z"
        >
          Apply
          <span
            class="hide-md css-6g5xt6"
          >
             
            Now
            <svg
              class="css-6832e"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M85 277.375h259.704L225.002 397.077 256 427l171-171L256 85l-29.922 29.924 118.626 119.701H85v42.75z"
              />
            </svg>
          </span>
        </button>
      </form>
    </div>
  </header>
</div>
`;

exports[`render as espected Layout 1`] = `
<div>
  <div
    class="css-1bus680-Wrapper e1txfvao1"
  >
    <div
      class="css-10i4iws-TopBar e1txfvao0"
    >
      <img
        alt="GLD Funding logo"
        class="css-1x09otm"
        src="./static/images/logo-white.svg"
      />
      <div
        class="show-md mb-open-btn css-l6kea7-Open esgq4xm3"
      >
        <div
          class="css-1s4dz1f"
        />
        <div
          class="css-12ptazb"
        />
        <div
          class="css-1s4dz1f"
        />
      </div>
      <nav
        class="css-1dcz2ie-Nav esgq4xm1"
      >
        <svg
          class="show-md css-un36pl-Close esgq4xm2"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 512 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M278.6 256l68.2-68.2c6.2-6.2 6.2-16.4 0-22.6-6.2-6.2-16.4-6.2-22.6 0L256 233.4l-68.2-68.2c-6.2-6.2-16.4-6.2-22.6 0-3.1 3.1-4.7 7.2-4.7 11.3 0 4.1 1.6 8.2 4.7 11.3l68.2 68.2-68.2 68.2c-3.1 3.1-4.7 7.2-4.7 11.3 0 4.1 1.6 8.2 4.7 11.3 6.2 6.2 16.4 6.2 22.6 0l68.2-68.2 68.2 68.2c6.2 6.2 16.4 6.2 22.6 0 6.2-6.2 6.2-16.4 0-22.6L278.6 256z"
          />
        </svg>
        <div
          class="css-4cffwv"
        >
          <a
            class="esgq4xm0 css-1m8yfox-HL"
            href="./"
          >
            About
          </a>
          <a
            class="esgq4xm0 css-1m8yfox-HL"
            href="./"
          >
             What is MCA?
          </a>
          <a
            class="esgq4xm0 css-1m8yfox-HL"
            href="./"
          >
            Apply
          </a>
          <a
            class="esgq4xm0 css-1m8yfox-HL"
            href="./"
          >
            Contact
          </a>
        </div>
        <div
          class="css-4cffwv"
        >
          <a
            class="esgq4xm0 css-1m8yfox-HL"
            href="tel:1(877)498-4344"
          >
            Call Now 1 (877) 498-4344
          </a>
          <a
            class="show-md esgq4xm0 css-1m8yfox-HL"
            href="tel:1(877)498-4344"
          >
            Login
          </a>
          <a
            class="hide-md css-167ay55"
            href="tel:1(877)498-4344"
          >
            Login
          </a>
        </div>
      </nav>
    </div>
     hi 
    <div
      class="css-jnc4ax-Wrapper e18f8y721"
    >
      <div
        class="css-1tar3ql"
      >
        <div
          class="css-1bqr18x"
        >
          <img
            alt="McAfee SECURE"
            class="css-1uqn0ab-Footer"
            src="./static/images/mcafee.png"
          />
          <a
            class="e18f8y720 css-3euuug-HL"
            href="./"
          >
            Apply Now
          </a>
          <a
            class="e18f8y720 css-3euuug-HL"
            href="./"
          >
             What is MCA?
          </a>
          <a
            class="e18f8y720 css-3euuug-HL"
            href="./"
          >
            About us
          </a>
          <a
            class="e18f8y720 css-3euuug-HL"
            href="./"
          >
            Contact
          </a>
          <a
            class="e18f8y720 css-3euuug-HL"
            href="./"
          >
            Login
          </a>
        </div>
        <div
          class="css-ipt8zy"
        >
          <a
            class="e18f8y720 css-1smhtko-HL"
            href="https://www.instagram.com/gldfunding/"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              />
            </svg>
          </a>
          <a
            class="e18f8y720 css-1smhtko-HL"
            href="https://www.facebook.com/GLD-Funding-102567247876620/"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 320 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
              />
            </svg>
          </a>
          <a
            class="e18f8y720 css-1smhtko-HL"
            href="./"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
          </a>
          <a
            class="e18f8y720 css-1smhtko-HL"
            href="./"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 576 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`render as espected Message 1`] = `
<div>
  <section
    class="e7rx4o50 css-1ep92rf-Wrapper"
  >
    <img
      alt="mountains background"
      class="e7rx4o51 css-gn5u38-Background"
      src="./static/images/footer-image.jpg"
    />
    <div
      class="css-12y9y0h"
    >
      <div
        class="css-qqmf3e"
      >
        <div
          class="react-multi-carousel-list   css-jpalgg-StyledCarousel e132dzpo0"
        >
          <ul
            class="react-multi-carousel-track "
            style="transition: none; overflow: unset; transform: translate3d(0px,0,0);"
          />
        </div>
      </div>
      <div
        class="css-1s1r0fc"
      >
        <form
          class="css-148ky1p"
        >
          <label
            class="css-srhed4-Message"
            for="email"
          >
            email
          </label>
          <input
            class="css-m3t4fg"
            name="email"
            placeholder="email"
            required=""
            type="email"
          />
          <label
            class="css-srhed4-Message"
            for="message"
          >
            Message
          </label>
          <input
            class="css-m3t4fg"
            name="message"
            placeholder="message"
            required=""
          />
          <button
            class="css-ae7qj"
          >
            SEND MESSAGE
          </button>
          <p
            class="css-c6o2eq"
          >
            Do you have a question? 
            <br />
            Contact Us.
          </p>
        </form>
      </div>
    </div>
  </section>
</div>
`;

exports[`render as espected Modal 1`] = `<div />`;

exports[`render as espected Steps 1`] = `
<div>
  <section
    class="css-146wbcb-Wrapper ews2q5x0"
  >
    <div
      class="css-18e384y"
    >
      <div
        class="css-kg0zaz"
      >
        <div
          class="css-11p359h-VideoThumb ews2q5x2"
        >
          <img
            alt="video placeholder"
            class="ews2q5x1 css-1gf4r9w-Background"
            src="./static/images/demo.jpg"
          />
          <button
            class="ews2q5x3 css-18obo5a-PlayButton"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z"
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        class="css-1bm6goi"
      >
        <div
          class="css-3uy3bb"
        >
          <span
            class="css-19uslfw-Steps"
          >
            <div
              class="css-1b283uw"
            >
              1
            </div>
          </span>
          <div
            class="css-wnoyok"
          >
            <h1
              class="css-o1639m"
            >
              Apply
            </h1>
            <p
              class="css-eanlim"
            >
              Fill out an application today with your basic information. Once filled out, submit along with your last 3 months of your business bank statements.
            </p>
          </div>
        </div>
        <div
          class="css-3uy3bb"
        >
          <span
            class="css-lfc1pm-Steps"
          >
            <div
              class="css-1b283uw"
            >
              2
            </div>
          </span>
          <div
            class="css-wnoyok"
          >
            <h1
              class="css-o1639m"
            >
              Approve
            </h1>
            <p
              class="css-eanlim"
            >
              Our underwriting team will review your application and bank statements to develop a solution specific to your business needs. They will then contact you and discuss your offer. Repayments are flexible; there are no personal guarantees and minimal documents required.
            </p>
          </div>
        </div>
        <div
          class="css-3uy3bb"
        >
          <span
            class="css-1qkclsg"
          >
            <div
              class="css-1b283uw"
            >
              3
            </div>
          </span>
          <div
            class="css-wnoyok"
          >
            <h1
              class="css-o1639m"
            >
              Get Funded
            </h1>
            <p
              class="css-eanlim"
            >
              Once the contract is signed, you will receive your funds in your bank account the same day.
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
